Server/mods/deathmatch/logic/CScriptDebugging.cpp

```diff
@@ -15,6 +15,22 @@
 
 extern CGame* g_pGame;
 
+enum CScriptDebugging::DebugMessageLevels : std::uint32_t
+{
+    MESSAGE_TYPE_DEBUG = 0,
+    MESSAGE_TYPE_ERROR = 1,
+    MESSAGE_TYPE_WARNING = 2,
+    MESSAGE_TYPE_INFO = 3,
+    MESSAGE_TYPE_CUSTOM = 4,
+};
+
+enum CScriptDebugging::DebugScriptLevels : std::uint32_t
+{
+    ERRORS_ONLY = 1,
+    ERRORS_AND_WARNINGS = 2,
+    ALL = 3,
+};
+
```

Comment:
I think these enums should be in the header? For ```DebugMessageLevels``` you don't need to number the values ​​sequentially, as that will be the default

Reply:
They were previously in header, but botder suggested to move them to .cpp (to keep .h cleaner)

Reply:
Also, if you dont need to get the values of each enum field (like in a switch case) then use `enum class` instead of `enum`